qt/app.py

```diff
@@ -792,10 +802,9 @@ def _create_and_get_filesview_toolbar(self):
 
     def closeEvent(self, event):
         if self.shutdown.askBeforeQuit():
-            msg = _('If you close this window, Back In Time will not be able '
-                    'to shut down your system when the snapshot is finished.')
-            msg = msg + '\n'
-            msg = msg + _('Do you really want to close it?')
+            msg = _('If you close this window Back In Time will not be able '
+                    'to shut down your system when the snapshot has finished.'
+                    '\nDo you really want to close?')
```

Comment:
There shouldn't be any escape characters inside a translatable string. Because of that I separated that two paragraphs. Looks ugly in code but is better for translators using our Weblate translation platform and often not having technical background.

Reply:
This should not have been changed. I am not sure what happened here. I did not try to make any changes in this part of code. Probably a mistake.

Reply:
I thought the same. Then I assuming that your dev branch is outdated and not in sync with the current upstream dev branch.